packages/web-api/src/types/request/conversations.ts

```diff
@@ -31,10 +31,14 @@ export interface IsPrivate {
   /** @description Whether the channel should be private. */
   is_private?: boolean;
 }
-interface Message extends Channel {
+interface MessageSpecifier extends Channel {
   /** @description Unique identifier of message. */
   ts: string;
 }
+interface Message {
+  /** @description A message to send to the user who requested the invite. */
+  message?: string;
```

Comment:
I could easily fold this into the `deny` method's arguments directly; the reason it is its own interface right now is that I factored it out when, earlier in these APIs' pilot, there were other methods (that the team decided not to ship) that used the same property, so instead of repeating it across different API argument interfaces, I factored it out and reused them. But, I just removed all those other APIs since they are not public yet.

I can fold that back into just the Deny arguments - let me know!

Reply:
No need: I trust your judgement! At first glance, it's just surprising that a `Message` interface wouldn't feature the `message` prop. 😂 

Carry on!